MercadoPagoSDK/MercadoPagoSDK/Flows/OneTap/OneTapFlowModel.swift

```diff
@@ -311,6 +313,38 @@ internal extension OneTapFlowModel {
 
         return savedCardWithESC
     }
+    
+    func needCreateOptionalToken() -> Bool {
+        guard let paymentMethod = self.paymentData.getPaymentMethod() else {
+            return false
+        }
+
+        guard let paymentOptionSelected = paymentOptionSelected else {
+            return false
+        }
+
+        if !readyToPay {
+            return false
+        }
+        
+        if paymentData.hasToken() {
+           return false
+        }
```

Comment:
Since all conditionals return the same value, maybe grouping them together would be nice.
```suggestion
        guard let paymentMethod = self.paymentData.getPaymentMethod(),
        let paymentOptionSelected = paymentOptionSelected,
        readyToPay,
        !paymentData.hasToken() else {
          return false
        }
```

or 

```suggestion
        guard let paymentMethod = self.paymentData.getPaymentMethod(),
        let paymentOptionSelected = paymentOptionSelected else {
          return false
        }
        
        if !readyToPay, paymentData.hasToken() {
          return false
        }
```

Reply:
Good suggestion, tested locally and pushed recently 👍🏻